notebooks/05-operations/01-druid-logs-configuration.ipynb

```diff
@@ -0,0 +1,524 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c9fc0614-4eec-4c39-ad22-ea10afa00e9d",
+   "metadata": {},
+   "source": [
+    "# Apache Druid logging\n",
+    "\n",
+    "Log configuration and maintenance is an essential task for operators of an Apache Druid cluster. Developers makes extensive use of Log4J throughout Druid to emit information that's useful for day-to-day monitoring and for troubleshooting. They not only enable you to investigate issues and solve problems, but to understand how each of Druid's processes work in isolation and in collaboration with one another.\n",
+    "\n",
+    "In this notebook you will take a tour of the out-of-the-box Log4J configuration in Apache Druid, and use some simple terminal commands to locate them and look at their contents."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd9a55db-61ff-4fa4-9655-2fef292ed2aa",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "This tutorial works with Druid 29.0.0 or later.\n",
+    "\n",
+    "Launch this tutorial and all prerequisites using the `jupyter` profile of the Docker Compose file for Jupyter-based Druid tutorials. For more information, see the Learn Druid repository [readme](https://github.com/implydata/learn-druid).\n",
+    "\n",
+    "__DO NOT__ use the `jupyter-druid` profile with this tutorial as it will conflict with your locally running copy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "151ede96-348f-423c-adc7-0e23c6910b9e",
+   "metadata": {},
+   "source": [
+    "## Initialization\n",
+    "\n",
+    "To use this notebook, you will need to run Druid locally.\n",
+    "\n",
+    "You will also make extensive use of the terminal, which you can place alongside this notebook or on another screen.\n",
+    "\n",
+    "### Install tools\n",
+    "\n",
+    "Open a local terminal window.\n",
+    "\n",
+    "If you have not already install `wget` or `multitail`, run the following commands to install these tools using `brew`.\n",
+    "\n",
+    "```bash\n",
+    "brew install multitail ; brew install wget\n",
+    "```\n",
+    "\n",
+    "Run the following command to pull the default configuration for `multitail` to your home folder. Do not run this command if you are already running `multitail` as it will overwrite your own configuration.\n",
```

Comment:
```suggestion
    "To fetch the default configuration for `multitail` to your home folder, execute the following command. Skip this step if you are already running `multitail` as it will overwrite your own configuration.\n",
```